VISION.md

```diff
@@ -51,25 +51,25 @@ In the past, the OpenEBS project has tried to accomplish too much. Our scope was
 | 9. | ```Rest API``` and ```Cli``` for accessing all capabilities |
 | 10. | Storage Reporting, alerting, monitoring, metrics |
 | 11. | Read/Write Access Modes for RWO, ROX, RWOP - (*RWX with strict limitations) |
-| 12. | Deployable as ```On-premise```, ```SaaS/PaaS in-cloud```, ```Bare Metal```, ```HyperVisor``` infrastructure |
+| 12. | Deployable as ```On-premise``, ```SaaS/PaaS in-cloud``, ```Bare Metal```, ```HyperVisor``` infrastructure |
```

Comment:
Hmmmm.  Should probably be "Bare metal, HyperVisor,Hosted/Cloud-based infrastructure"

Reply:
HyperVisor --> Hypervisor

Reply:
@edrob999 You want me to remove the terms `On-premise, SaaS/PaaS in-cloud` and change that to only `Bare metal, HyperVisor,Hosted/Cloud-based infrastructure`?